tools/confix/data/v0.50-app.toml

```diff
@@ -66,6 +70,10 @@ iavl-cache-size = 781250
 # Default is false.
 iavl-disable-fastnode = false
 
+# IAVLLazyLoading enable/disable the lazy loading of iavl store.
```

Comment:
This should not be re-added.

Reply:
was thinking the same but confused why it got generted